optimum/habana/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -269,6 +270,7 @@ def gaudi_gpt_neox_model_forward(
     return_dict: Optional[bool] = None,
     cache_position: Optional[torch.LongTensor] = None,
     token_idx: Optional[torch.Tensor] = None,
+    **flash_attn_kwargs: Unpack[FlashAttentionKwargs],
```

Comment:
```suggestion
    **kwargs,
```
Let's just add kwargs here and remove the import, we don't need `FlashAttentionKwargs`.